apps/appointments/serializers.py

```
@@ -0,0 +1,60 @@
+            start_time=validated_data['start_time'],
+            end_time=validated_data['end_time'],
+            notes=validated_data.get('notes', ''),
+            delivery_address=delivery_address or None,
+        )
+        if validated_data.get('notes'):
+            appointment.notes = validated_data['notes']
+        return appointment
+
+    def validate(self, attrs):
+        if attrs['end_time'] <= attrs['start_time']:
+            raise serializers.ValidationError('End time must be after start time.')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)